tls/s2n_ecc_preferences.c

```diff
@@ -69,3 +69,15 @@ int s2n_check_ecc_preferences_curves_list(const struct s2n_ecc_preferences *ecc_
     return S2N_SUCCESS;
 }
 
+/* Determines if query_iana_id corresponds to a curve for these ECC preferences. */
+bool s2n_ecc_preferences_includes(const struct s2n_ecc_preferences *ecc_preferences, uint16_t query_iana_id) {
+    notnull_check(ecc_preferences);
```

Comment:
I just realized I'm using `notnull_check()` in a function that returns `bool`. While apparently syntactically correct, it sidesteps my intention of causing an error when `ecc_preferences == NULL`.